packages/playground-server/src/db.ts

```diff
@@ -276,7 +276,8 @@ export async function createUser(user: User): Promise<User> {
     email: user.attributes.email,
     eth_address: user.attributes.ethAddress,
     multisig_address: user.attributes.multisigAddress,
-    node_address: user.attributes.nodeAddress
+    node_address: user.attributes.nodeAddress,
+    transaction_hash: ""
```

Comment:
Why is `transaction_hash` on the user model? Also its a pretty generic token. Maybe specify as `multisig-deployment_transaction_hash`? In either case, seems odd to be on the user model.

Reply:
@snario It probably doesn't need to be here, as it is quickly discarded. But I'm thinking - what if the transaction gets interrupted for some reason? We could use it to resume and finish the account creation flow.